docs/pf-keys/python/getting_started.rst

```diff
@@ -93,6 +93,8 @@ Parflow run arguments:
 
       --write-yaml          Enable config to be written as YAML file
 
+      --validation-error    Only print validation results for key/value pairs with errors
```

Comment:
Should this be the default behavior?   So users are more likely to see validation errors and not have as much to look at in the normal case.

Reply:
I've discussed this with Laura and Reed and they had slightly different opinions on which should be the default. If you feel strongly that `--validation-error` should be default, I am happy to make that change, and I believe @lecondon and @reedmaxwell would approve as well.

Reply:
Will do the printing for now.  Will discuss with Larua and Reed.

Reply:
He ended up changing this option to [`--validation-verbose`](https://github.com/parflow/parflow/pull/279/files#diff-175eb4c2a4254038a880ab2f7f077019b97a088ae5d2c452e725b856d58e6c4aR96). So now by default it only prints validation results for errors, but will print all validation results with `--validation-verbose`.